src/example_main.cpp

```diff
@@ -0,0 +1,11 @@
+#include <memory>
+
+#include "i_comm.hpp"
+
+int main(int argc, char **argv) {
+
+    std::unique_ptr<i_comm> comm = std::make_unique<rawcan_comm>(0x100, "can0");  
+    roboteq_controller::roboteq_controller motor_controller = roboteq_controller(std::move(comm)); 
```

Comment:
since copy construction is invalid this could either be move assignment or just a constructor call:

Move assignment:
` roboteq_controller::roboteq_controller motor_controller = std::move(roboteq_controller(std::move(comm))); `

Normal Constructor:
 `roboteq_controller::roboteq_controller motor_controller(std::move(comm)); `

While both are valid, you should use the normal constructor because it cleaner and technically more efficient.

Reply:
![image](https://user-images.githubusercontent.com/56097508/69999751-5c9e2f80-1527-11ea-83d2-363801f18388.png)

Reply:
yup